test/built-ins/Temporal/PlainDate/prototype/since/order-of-operations.js

```diff
@@ -86,8 +86,8 @@ const instance = new Temporal.PlainDate(2000, 5, 2, ownCalendar);
 
 const otherDatePropertyBag = TemporalHelpers.propertyBagObserver(actual, {
   year: 2001,
-  month: 5,
-  monthCode: 'M05',
+  month: 6,
+  monthCode: "M06",
```

Comment:
Which fast path is this avoiding?

Reply:
You can see it a bit farther down in the same commit, where I added a new test case with the previous value of `otherDatePropertyBag`. In RoundDuration with smallestUnit = year, if there are no excess months/weeks then the dateUntil call will be skipped in DifferenceDate. This change adds an excess month.